app/api/domains/osu.py

```diff
@@ -434,20 +434,12 @@ async def lastFM(
     """
 
 
-# bancho.py supports cheesegull mirrors, chimu.moe and nasuya.xyz.
-# chimu.moe and nasuya.xyz handle things a bit differently than cheesegull,
-# and has some extra features we'll eventually use more of.
-USING_CHIMU = "chimu.moe" in app.settings.MIRROR_URL
-USING_NASUYA = "nasuya.xyz" in app.settings.MIRROR_URL
-
-DIRECT_SET_ID_SPELLING = "SetId" if USING_CHIMU else "SetID"
-
 DIRECT_SET_INFO_FMTSTR = (
-    "{{{setid_spelling}}}.osz|{{Artist}}|{{Title}}|{{Creator}}|"
-    "{{RankedStatus}}|10.0|{{LastUpdate}}|{{{setid_spelling}}}|"
-    "0|{{HasVideo}}|0|0|0|{{diffs}}"  # 0s are threadid, has_story,
+    "{SetID}.osz|{Artist}|{Title}|{Creator}|"
```

Comment:
@minisbett does this work with chimu? (did they fix the inconsistency?)

Reply:
yes I talked with kotrik and he fixed it